### PR TITLE
[ADD]: copy to clipboard extension method on String

### PR DESCRIPTION
**Pull request by baimamboukar**
Added copy to clipboard extension method on String 

@AbhishekDoshi26

### CODE REVIEW

File: lib/src/super_extensions_base.dart [hunk 1]
```diff
@@ -136,14 +137,36 @@ extension StringExtension on String {
...
+  /// This is how you can use it:
+  /// ```dart
+  /// String text = 'The beautiful thing with Dart is that you can create your own extensions.';
+  ///
+  /// TextButton(
+  /// child: Text('Copy to Clipboard'),
+  ///  onPressed: (){
+  ///   text.copyToClipboard();
+  /// },
+  /// );
```
Comment (AbhishekDoshi26) [inline lib/src/super_extensions_base.dart]: Alignment should be proper.

File: lib/src/super_extensions_base.dart [hunk 2]
```diff
@@ -136,14 +137,36 @@ extension StringExtension on String {
...
+  ///
+  /// ```
+  void copyToClipboard() {
+    Clipboard.setData(
+      ClipboardData(
+        text: this,
+      ),
+    );
+  }
```
Comment (AbhishekDoshi26) [inline lib/src/super_extensions_base.dart]: Use arrow operator.

**Review by AbhishekDoshi26 — CHANGES_REQUESTED** [review]

File: lib/src/super_extensions_base.dart (hunk 2 above)
Comment (AbhishekDoshi26) [reply lib/src/super_extensions_base.dart]: @baimamboukar

File: lib/src/super_extensions_base.dart (hunk 1 above)
Comment (baimamboukar) [reply lib/src/super_extensions_base.dart]: Okay! noted

File: lib/src/super_extensions_base.dart (hunk 2 above)
Comment (baimamboukar) [reply lib/src/super_extensions_base.dart]: Alright Sir! Doing it now.... 👍🏾

**Comment by baimamboukar** [comment]
Hello sir @AbhishekDoshi26 ! The requested changes have been added ❇️✅

**Review by AbhishekDoshi26 — APPROVED** [review]

**Comment by AbhishekDoshi26** [comment]
Perfect. @baimamboukar waiting for the github actions to complete. Nice work 🚀

**Pull request merged**